src/site/content/en/testing/a-glance-in-test-automation/index.md

```diff
@@ -0,0 +1,122 @@
+---
+layout: post
+title: Quick overview of types of test automation
+date: 2023-04-30
+description: |
+  The web app manifest is a simple JSON file that tells the browser about your
+  web application and how it should behave when installed on the user's mobile
+  device or desktop.
+tags:
+  - test automation
+---
+
+We've all been there: what is a recurring coding meme that happens all too often in real life?
+
+{% Img src="image/placeholder.jpg", alt="A cupboard with two drawers you cannot open at the same time", width="380", height="240" %}
```

Comment:
I don't know how to add images as an external contributor, so I will use placeholders for now. How is the process in my case?